lib/tic_tac_toe.ex

```diff
@@ -1,29 +1,33 @@
 defmodule TicTacToe do
 
-  def start do
-    run_application(true)
-  end
+  def start, do: run_application(true)
 
-  def run_application(false), do: stop
-  def run_application(true) do
-    board = [1,2,3,4,5,6,7,8,9]
-    play(board, Board.game_over?(board))
-    run_application(Ui.play_again?)
+  defp run_application(false), do: stop
+  defp run_application(true) do
+    empty_board = [1,2,3,4,5,6,7,8,9]
```

Comment:
The fact that an empty board is a an array from 1 to 9 is a concern of the board module. I'd create a function there